aws/components/globaldb/state.ftl

```diff
@@ -0,0 +1,51 @@
+[#ftl]
+
+[#macro aws_globaldb_cf_state occurrence parent={} ]
+    [#local core = occurrence.Core]
+    [#local solution = occurrence.Configuration.Solution]
+
+    [#local id = formatResourceId(AWS_DYNAMODB_TABLE_RESOURCE_TYPE, core.Id )]
```

Comment:
Table names all need to include product environment and segment (if not default) to ensure uniqueness. Think this is something we should automatically add.

From a coding perspective, might also be useful to know what this prefix is, though if using linking, then the table name is one of the attributes so they should really care.

Reply:
Current implementation was using the randomly generated CFN name, for the globaldb I've added naming support using core.FullName 

Prefix isn't really part of this since we give them the table and it has the full name